app/MyApp/app/Home-Urdu.css

```diff
@@ -1150,9 +1150,9 @@ textarea[name="comment"]
     margin-left: 20px;
     width:auto;
 }
-.form .btn {
+/*.form .btn {
     margin: 5px 20px 20px 0px;
-}
+} */
```

Comment:
+/*.form .btn {
      margin: 5px 20px 20px 0px;
 -}
 +} */
comment should be removed